cdap-proto/src/main/java/co/cask/cdap/proto/DefaultThrowable.java

```diff
@@ -26,6 +26,13 @@
   private final StackTraceElement[] stackTraces;
   private final DefaultThrowable cause;
 
+  public DefaultThrowable(String className, String message, StackTraceElement[] stackTraces, DefaultThrowable cause) {
```

Comment:
Would be good to add a comment on when this is used vs when the other constructor is used.